__tests__/spec/migrate.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-env jest */
+
+const path = require('path')
+const fs = require('fs')
+const fse = require('fs-extra')
+const { spawn } = require('../../lib/exec')
+const os = require('os')
+const { mkdtempSync } = require('../util')
+const { sleep } = require('../../lib/utils')
+
+const projectDirectory = path.join(mkdtempSync(), 'migrate-checks')
+const appDirectory = path.join(projectDirectory, 'app')
+const assetsDirectory = path.join(appDirectory, 'assets')
+const fixtureProjectDirectory = path.join(__dirname, '..', 'fixtures', 'test-prototype')
+const cliPath = path.join(__dirname, '..', '..', 'bin', 'cli')
+
+const pkg = {
+  name: 'govuk-prototype-kit',
+  version: '11.0.0',
+  dependencies: {
+    'govuk-frontend': '^4.3.1',
+    'govuk-prototype-kit': 'file:../../..'
+  }
+}
+
+describe('migrate test prototype', () => {
+  beforeAll(async () => {
+    fse.copySync(fixtureProjectDirectory, projectDirectory, { clobber: true })
+    fse.writeJsonSync(path.join(projectDirectory, 'package.json'), pkg, { clobber: true })
```

Comment:
I wonder if we can avoid having to do this and also make our test more complete if we include a copy of the v11 prototype release archive in the repo, which we extract when it is time to do the test?